src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -242,7 +243,19 @@ def test_acckernelsdirective_gencode(default_present):
         "      END DO\n"
         "      !$acc end kernels\n" in code)
 
-# Class ACCKernelsDirective end
+
```

Comment:
Since we're here would you mind changing lines 240-241 to use an f-string since that's the only one in this file?

Reply:
Done